src/client/tile.cpp

```diff
@@ -2266,7 +2265,7 @@ video::ITexture* TextureSource::getNormalTexture(const std::string &name)
 	if (isKnownSourceImage("override_normal.png"))
 		return getTexture("override_normal.png");
 	std::string fname_base = name;
-	std::string normal_ext = "_normal.png";
+	static const std::string normal_ext = "_normal.png";
```

Comment:
because it's just a variable not used anywhere except here and it's a function called often by client